File: VueClient/src/models/Location.ts
export default class Location {
  public Id: number;
  public LocationName: string;
  public Latitude: string;
  public Longitude: string;

  constructor(
    id?: number,
    locationName?: string,
    latitude?: string,
    longitude?: string
  ) {
    this.Id = id || 0;
    this.LocationName = locationName || "";
    this.Latitude = latitude || "";
    this.Longitude = longitude || "";
  }
}

File: ApiRest/Models/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ApiRest.Models
{
    public class Location
    {
        public int LocationId { get; set; }
        public string LocationName { get; set; }
        public string Latitude { get; set; }
        public string Longitude { get; set; }
        public User User { get; set; }
    }
}